docs/l10n.rst

```diff
@@ -163,11 +163,11 @@ E.g: ``https://addons-dev.allizom.org/en-US/firefox/`` becomes
 ``https://addons-dev.allizom.org/dbr/firefox/``.
 
 
-.. _`dev`: https://addons-dev.allizom.org
-.. _`stage`: https://addons.allizom.org
+.. _`dev`: https://addons-dev.allizom.org/
+.. _`stage`: https://addons.allizom.org/
 .. _`potools`: https://github.com/mozilla/potools/
 .. _`Pontoon`: https://pontoon.mozilla.org/
 .. _`addons-server on Pontoon`: https://pontoon.mozilla.org/projects/amo/info/
 .. _`addons-frontend on Pontoon`: https://pontoon.mozilla.org/projects/amo-frontend/info/
-.. _`new profile`: https://support.mozilla.org/kb/profile-manager-create-and-remove-firefox-profiles
+.. _`new profile`: https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles
```

Comment:
Should this have the locale removed to allow for redirs based on accept-language?

Reply:
Good catch!

Reply:
4c95ce9